client/config/webpack.config.dev.js

```diff
@@ -12,6 +12,7 @@ module.exports = merge(webpackCommonConfig, {
   output: {
     path: path.join(__dirname, '/dist/'),
     filename: 'bundle.js',
+    chunkFilename: '[name].bundle.js',
```

Comment:
Also, let this be `[name].[hash].js` or `[hash].bundle.js`.. I don't think we really need the name here

Reply:
@BolajiOlajide if that's better I'll make the changes. I simply followed web pack documentation.

Reply:
Yeah! It's better so in the event the browser caches any of the bundles and it changes, the browser will see the new update instead of pointing to the old one.

If we make use of the name, in the event the browser caches the response from a particular name and it isn't much different, the browser might not display the new changes that's why I suggested you add the hash to it too.